spec/ace.md

```diff
@@ -49,6 +51,14 @@ Volumes that are specified in the Pod Manifest are mounted into each of the apps
 For example, say that the worker-backup and reduce-worker both have a `mountPoint` named "work".
 In this case, the executor will bind mount the host's `/opt/tenant1/work` directory into the `path` of each of the matching "work" `mountPoint`s of the two app filesystems.
 
+If the target `path` does not exist in the rendered filesystem, it SHOULD be created, including any missing parent directories. If the target `path` is a non-empty directory, its contents SHOULD be discarded (e.g. obscured by the bind mount). If the target `path` refers to a file, ACE SHOULD remove that file and create a directory in its place.
+
+Mount point `path` directories that ACE creates SHOULD be owned by UID 0 and GID 0, and have access mode `0755` (`rwxr-xr-xr-x`). ACE implementation MAY provide a method for administrator to specify different permissions on a per-pod basis.
```

Comment:
I'm concerned about consistency: if we reverse the ordering, this means that the path can be created with any permissions, which can give different results on different implementations (or even in different installations of the same implementation, depending on GID or umask of ACE process).

Consistency of the results can be partially circumvented by just including empty directory in the ACI, but this won't help with mount points introduced by app override in pod manifest: there's no way to make these behave consistently without specifying suggested permissions.

We can also simply say that it's an error if the mount path doesn't exist, but I think that would be too strict: flexibility of mounting volumes anywhere via app overrides seems to be a feature rather than a bug.